### PR TITLE
Added information about timezones in startDateTime and endDateTime

### LINKED ISSUE #4359: Clarify Request start/end DateTime timezone info
There's a lot about the response providing info on the timezone, but not for the request for filtering.  It appears if no timezone is given on the provided query element, that UTC is assumed?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 18fec655-72d2-4bdf-9201-ccfc37848e1e
* Version Independent ID: 63536b57-171d-f44f-97db-78afa8b499ae
* Content: [List calendarView - Microsoft Graph v1.0](https://docs.microsoft.com/en-us/graph/api/user-list-calendarview?view=graph-rest-1.0)
* Content Source: [api-reference/v1.0/api/user-list-calendarview.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/master/api-reference/v1.0/api/user-list-calendarview.md)
* Product: **microsoft-identity-platform**
* Technology: **microsoft-graph**
* GitHub Login: @dkershaw10
* Microsoft Alias: **MSGraphDocsVteam**

### PR DESCRIPTION
**Pull request by jasonjoh**
Fixes #4359

### CODE REVIEW

File: api-reference/beta/api/user-list-calendarview.md [hunk 1]
```diff
@@ -51,14 +51,16 @@ GET /users/{id | userPrincipalName}/calendarGroups/{id}/calendars/{id}/calendarV
...
-|:---------------|:--------|:----------|
-|startDateTime|String|The start date and time of the time range, represented in ISO 8601 format. For example, "2015-11-08T19:00:00.0000000".|
-|endDateTime|String|The end date and time of the time range, represented in ISO 8601 format. For example, "2015-11-08T20:00:00.0000000".|
+| Parameter     | Type   | Description                                                                                                            |
+|:--------------|:-------|:-----------------------------------------------------------------------------------------------------------------------|
+| startDateTime | String | The start date and time of the time range, represented in ISO 8601 format. For example, "2019-11-08T19:00:00-08:00". |
+| endDateTime   | String | The end date and time of the time range, represented in ISO 8601 format. For example, "2019-11-08T20:00:00-08:00".   |
+
+The values of `startDateTime` and `endDateTime` are interpreted using the timezone offset specified in the value and are not impacted by the value of the `Prefer: outlook.timezone` header if present. If no timezone offset is included in the value, it is interpreted as UTC.
```
Comment (angelgolfer-ms) [inline api-reference/beta/api/user-list-calendarview.md]: Line 59: I have observed this too.

File: api-reference/beta/api/user-list-calendarview.md [hunk 2]
```diff
@@ -51,14 +51,16 @@ GET /users/{id | userPrincipalName}/calendarGroups/{id}/calendars/{id}/calendarV
...
 
 This method also supports some of the [OData Query Parameters](https://developer.microsoft.com/graph/docs/concepts/query_parameters) to help customize the response.
 
-> [!NOTE] 
+> [!NOTE]
 > The **createdDateTime** and **lastModifiedDateTime** properties of [event](../resources/event.md) do not support `$select`. To get their values, simply query on **calendarView** without applying `$select`.
 
 ## Request headers
```
Comment (angelgolfer-ms) [inline api-reference/beta/api/user-list-calendarview.md]: GitHub wouldn't let me place the next comment closer to line 89, so here it is - please also check and fix the start and end time values provided in the example request. Here in this topic, they are not correct either.

File: api-reference/v1.0/api/user-list-calendarview.md [hunk 3]
```diff
@@ -83,7 +85,7 @@ Here is an example of the request.
   "name": "user_get_calendarview"
 }-->
 ```msgraph-interactive
-GET https://graph.microsoft.com/v1.0/me/calendarView?startDateTime=2016-01-01T19:00:00.0000000&endDateTime=2016-10-01T19:00:00.0000000 
+GET https://graph.microsoft.com/v1.0/me/calendarView?startDateTime=2016-01-01T19:00:00.0000000&endDateTime=2016-10-01T19:00:00.0000000
```
Comment (angelgolfer-ms) [inline api-reference/v1.0/api/user-list-calendarview.md]: Please also fix the start and end time values in the example request.

**Comment by jasonjoh** [comment]
@angelgolfer-ms sure I'll fix those too. I just went by the TOC in Calendar which took me to the user topics :).

File: api-reference/beta/api/user-list-calendarview.md (hunk 2 above)
Comment (jasonjoh) [reply api-reference/beta/api/user-list-calendarview.md]: Fixed this.

File: api-reference/v1.0/api/user-list-calendarview.md (hunk 3 above)
Comment (jasonjoh) [reply api-reference/v1.0/api/user-list-calendarview.md]: Fixed this too

**Pull request merged**